### ISSUE TITLE
Internal names acces does not use x0020

### ISSUE THREAD

**Issue by Biggum82** [SharePoint/sp-dev-docs#7058]
Hello. I'm following these series of articles to pratice due to some future project I'm planning to do. I think they are very useful.

In this example is used a replacement for sapces using text "_x0020_" to acces the List Column.
And it is quite well explained.
This is not working.

The actual method that works is just erase spaces and concatenate all caracters in a string.
In the example, the resulting string for "Added to Corporate DB" field is "AddedtoCorporateDB".

I have found this solution googling around, not in MS Docs.

1    Open the List Settings page
 2   Under the Columns section, select a column to view the Edit Column page
 3   The URL of this page includes the internal name in the query string. For example, the URL for the "Added to Corporate DB" field includes the following:

https://{URL_TENANT}/sites/{developer_site}/hongkong/_layouts/15/FldEdit.aspx?List=%7BED420F68%2DEA01%2D40D8%2DBDDD%2D44923AB1E4BE%7D&Field=AddedtoCorporateDB 

The value for the Field parameter, **AddedtoCorporateDB** at the end of the URL, is the internal name for Created By.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 2b2f2c15-847c-4041-676d-ce5693354660
* Version Independent ID: 7a845c03-a74a-4cfe-7347-3984fb795856
* Content: [Add SharePoint write operations to the provider-hosted add-in](https://docs.microsoft.com/en-us/sharepoint/dev/sp-add-ins/add-sharepoint-write-operations-to-the-provider-hosted-add-in)
* Content Source: [docs/sp-add-ins/add-sharepoint-write-operations-to-the-provider-hosted-add-in.md](https://github.com/SharePoint/sp-dev-docs/blob/master/docs/sp-add-ins/add-sharepoint-write-operations-to-the-provider-hosted-add-in.md)
* Product: **sharepoint**
* GitHub Login: @VesaJuvonen
* Microsoft Alias: **vesaj**

**Comment by bcameron1231** [comment]
A brief history, SharePoint classic would always replace spaces with _x0020_ in your internal column names. However, this is not always the case, especially in Modern, where you can create columns on the list view itself, and it will not do this, it will just concatenate the values like you see here.

So while in some scenarios, it will not add the _x0020_ , there are still many cases where it will depending on how the column is created... so this document is still valid.

Long story short, whenever working with the SharePoint programatically, it's always best to confirm the internal value of the column like you did.
